sandbox/OPEN-SOURCE.md

```diff
@@ -0,0 +1,29 @@
+# Open Source
+
+Defense Unicorns values Open Source. We understand that Open Source is a way for all members of the company to prevent silos and work in a collaboration-first mentality. 
```

Comment:
For incorporating `free` into this we could explain it better potentially using the word freedom instead i.e. Free(dom) and Open Source or Open Source Freedom.  That is one issue with English being both libre and gratis are collapsed into a single word.

Reply:
I like the use of the word free or freedom here to differentiate. Would like to submit that I have difficulty personally valuing an object itself but am 100% on bought in on the `principles` that are at the heart of this particular object; the things that make it what it is. (i.e. Defense Unicorns values Free(dom) and Open Source Software `principles`). 

https://opensource.com/open-source-way

Reply:
Under the `#Why?` section, I propose we build an ordered list of FOSS principles that resonate with Defense Unicorns, each independently focusing on it's purpose / supporting value.

```
# Why?

Convergence & Collaboration: We believe that we are better together and these factors stand above all others to intensely accelerate mission outcomes. We relentlessly pursue alignment and partner where it exists to solve problems that no one can solve alone while enabling others to contribute in the future. We are on mission to destroy silos with the infectious culture of collaboration-first.

Transparency: We believe that thoughts, ideas and work should be accessible to build upon one another's contributions, formulate new ideas/improve existing ideas, make more effective decisions, understand how decisions affect us and iterate at speed.
```